Python/codecs.c

```diff
@@ -700,27 +700,38 @@ PyObject *PyCodec_IgnoreErrors(PyObject *exc)
 }
 
 
+static inline int
```

Comment:
Wouldn't it be better to move this inline function into the header file? That way it can omit the declaration in the header file and integrate well. It looks like a public API.

Reply:
In fact, I implemented https://github.com/python/cpython/pull/125201 which should replace this static inline function.